packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -405,3 +405,60 @@ instance:
   });
 </script>
 ```
+
+#### Tracking the Add2Cart event from PDP
+
+The X interface incorporates a mechanism that enables the tracking of the client's Add2Cart interaction by the shopper. The approach to achieving this varies depending on the website's structure.
+
+**NON-SPA**
+1. When initialising the script a new param `productId` should be added.
+
+```html
+window.InterfaceX.init({
+    instance: "instanceName",
+    lang: "es",
+    documentDirection: "ltr",
+    scope: t,
+    currency: "EUR",
+    consent: !0,
+    env: void 0,
+    isSPA: !0,
```

Comment:
My example in the task was wrong, I don't know where I copied those values but are not descriptive for developers that have to integrate this:

```suggestion
    scope: 'desktop',
    currency: "EUR",
    consent: true,
    isSPA: false,
```